boards/arm/lpcxpresso55s16/board.cmake

```diff
@@ -1,9 +1,14 @@
 #
 # Copyright (c) 2020 Henrik Brix Andersen <henrik@brixandersen.dk>
+# Copyright 2023 NXP
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 
+board_runner_args(linkserver  "--device=LPC55S16:LPCXpresso55S16")
 board_runner_args(jlink "--device=LPC55S16" "--reset-after-load")
+board_runner_args(pyocd "--target=lpc55s16")
 
+include(${ZEPHYR_BASE}/boards/common/linkserver.board.cmake)
```

Comment:
NXP BU has decided that the Linker Server must be the default runner. 
It is not my wish, as I prefer PyOCD ;)
Hi @DerekSnell, please comment the requirement to make LinkServer as a default one.

Reply:
Hi @butok ,
NXP is planning to make LinkServer the default runner on all NXP-supported boards in Zephyr.  This transition has not happened yet.  

For this PR, you have two choices:

1. Based on your testing with LinkServer on these boards, if you find the `west flash` and `west debug` commands perform well with LinkServer, you can update these boards to use LinkServer as the default runner.
2. Otherwise, leave JLink as the default runner, and NXP will change the default to LinkServer in the future.

Thank you for your contribution

Reply:
OK, LinkServer works for me on these platforms.

Reply:
@butok- is Linkserver currently the default platform with the changes in this PR?